### ISSUE TITLE
공유감사합니다. 구현이 그런데 다 되어 있는 것은 아닌거 맞나요?

### ISSUE THREAD

**Issue by Gogumee** [taki0112/Batch_Instance_Normalization-Tensorflow#1]
먼저 공유 감사합니다. 

In practice, we found that it is beneficial to increase the learning rate for updating ρ. It fairly matches
the theoretical aspect that the gradient of loss ` with respect to ρ tends to be small because it is
calculated by multiplying the difference between xˆ(B) and xˆ(I)

논문에 이 부분은 아직 구현 안된것 같은데 맞나요?
